src/css/profile/mobile/themes/dark.variables.less

```diff
@@ -41,8 +41,9 @@
     @button-background-contained:  @color-white;
     @text-input-invalid-color: #ff6666;
 
-    @dropdown-menu-options-border: #525252;
+    @dropdown-menu-options-border: 3 * @px_base solid #525252;
     @dropdown-menu-options-background: #3d3d3d;
+    @dropdown-menu-options-color: @color-white;
```

Comment:
How about using #number code instead of @ color-white to be consistent?

Reply:
I prefer variables to numbers  since they are more descriptive.
Also changed @dropdown-menu-options-color to @_black instead of #000000
in src/css/profile/mobile/themes/light.variables.less